src/ClusterBootstrap/ctl.py

```diff
@@ -316,6 +319,40 @@ def maintain_db(config, args):
         cursor.close()
 
 
+def cordon(config, args):
```

Comment:
Can these functions be reused by`ctl.py`?

Reply:
you mean cordon/uncordon? yes, as they are defined in this file. Currently this is the only place that they are invoked. if you mean reuse in deploy.py, then no. (check my reply below)